### ISSUE TITLE
Pull in cluster branch

### ISSUE THREAD

**Issue by glesica** [TravisWheelerLab/NINJA#29]
Pull in Robert Hubley’s cluster branch

George looked at this and it doesn’t look terrible but we probably ought to think about whether we want to do this first or last (one or the other seems right, doing it in the middle would just cause unnecessary chaos)

We should manually review the entire diff (couple hours, max) to try to maximize the odds that we don’t cause mysterious breakage

**Comment by jebrosen** [comment]
In trying to figure out #27 I discovered that while the `cluster` branch was based on another branch `develop`, `avx` is *not* based on `develop`. Aside from code changes some of the project structure was moved around, so this could become a bit tricky to review+merge.

**Comment by glesica** [comment]
Yeah, this is kind of a "nice to have" at this point. We're going to move certain changes forward and then take a look at the diff with the cluster branch and decide how we want to deal with it.

**Comment by traviswheeler** [comment]
A clarification: "this" means "pulling the cluster branch", as in "we view it as nice to have the cluster branch pulled into the upcoming NINJA release; if it doesn't happen right away, we'll figure out how to get it in there later"
